kinto/core/permission/postgresql/__init__.py

```diff
@@ -77,23 +136,12 @@ def initialize_schema(self, dry_run=False):
         with self.client.connect(readonly=True) as conn:
             result = conn.execute(query)
             if result.rowcount > 0:
-                logger.info('PostgreSQL permission schema is up-to-date.')
-                return
+                return 1
 
-        # Create schema
-        here = os.path.abspath(os.path.dirname(__file__))
-        sql_file = os.path.join(here, 'schema.sql')
-
-        if dry_run:
-            logger.info("Create permission schema from '{}'".format(sql_file))
-            return
-
-        # Since called outside request, force commit.
-        with open(sql_file) as f:
-            schema = f.read()
-        with self.client.connect(force_commit=True) as conn:
-            conn.execute(schema)
-        logger.info('Created PostgreSQL permission tables')
+        # Metadata table missing or has no
+        # permission_schema_version, and no user_principals table. We
+        # need to initialize.
+        return None
```

Comment:
nit: Ending a function without `return None` is the same as ending the function with a `return None`.

Reply:
While this is technically true, I believe returning `None` explicitly in this particular case is a better expression of intent here, since the function is meant to return None when it cannot identify the schema version in play. But I agree with your suggestion in a couple of other places.